packages/babel-core/src/transformation/file/options/build-config-chain.js

```diff
@@ -107,17 +110,31 @@ class ConfigChainBuilder {
 
     this.resolvedConfigs.push(loc);
 
-    const content = fs.readFileSync(loc, "utf8");
     let options;
+    if (path.extname(loc) === ".js") {
+      try {
+        const configModule = require(loc);
+        options = configModule && configModule.__esModule ? configModule.default : configModule;
```

Comment:
I updated this as per your suggestion @loganfsmyth (test is currently failing as a result). Is this correct? Because at this point it's just reading the JS config object, so it's now expecting the both `configModule.__esModule` and `configModule.default` to exist. Wondering if this is actually a real issue - even with babel-register, it will be reading the .babelrc.js file before transpilation, right? So I don't think we can actually have `import`s.

Apologies if this is a silly question - still getting familiar with how this works.